cypress/support/commands.ts

```diff
@@ -0,0 +1,78 @@
+/********************************************************************************
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+/* eslint-disable @typescript-eslint/no-namespace */
+
+export {}
+
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      login: (userType: string) => Chainable<void>
+      snackbarAlert: (msg: string) => Chainable<void>
+    }
+  }
+}
+
+// @ref https://on.cypress.io/custom-commands
+Cypress.Commands.add('login', (usertype) => {
+  // creating a session for user login
+  cy.session(
+    [usertype],
+    () => {
+      const companyName = Cypress.env('company').name
+      cy.visit(Cypress.env('baseUrl'))
+      // Perform login on Keycloak login page
+      cy.origin(Cypress.env('keycloak').centralUrl, () => {
```

Comment:
Without this change, I always had an error that the Company is undefined
```suggestion
      cy.origin(Cypress.env('keycloak').centralUrl, { args: { companyName } }, ({ companyName }) => {
```

Reply:
Okay.. Got it.. updated the code